src/main/java/com/spartronics4915/frc2020/subsystems/Launcher.java

```diff
@@ -126,11 +128,10 @@ public void rotateHood()
      * Rotates turret to a specific angle relative to the home position
      * @param absoluteAngle Angle in degrees you want to turn the turret relative to the home position
      */
-    public void turnTurret(double absoluteAngle)
+    public void turnTurret(Rotation2d absoluteAngle)
     {
-        // need enc or pot
-        // FIXME
-        // mTurretMotor.setPosition(absoluteAngle);
+        double output = mTurretPIDController.calculate(mTurretPotentiometer.get(), absoluteAngle.getDegrees());
+        mTurretMotor.setDutyCycle(output);
```

Comment:
surprised to see `setDutyCycle()` for positioning turret motor -- curious to see this working